### DISCUSSION TITLE
Good but no real support

### DISCUSSION THREAD

**Discussion by Duelbat** [spark-dotnet/framework#71] [General]
While it's all good and well to have a great "out of the box" framework, what about the support ? Looking around I see that the creator is nowhere to be seen! Support and general questions are largely ignored. Even a direct message to on Twitter / X was ignored. It's better to write your own project and know how it all fits togehter than to rely on non-existant support. I'm definitely out.

**Comment by harperjohn** [comment]
@Duelbat  I felt the same way... I tried contacting Wes using  a few ways, and he did (eventually) respond to my email message. It turns out that he had put so much time into this for so long that he just needed a break... but he said that he wanted to get back to it... But this was a few weeks ago and seems to have "gone dark" again...

I can definitely sympathize with the feeling of being "stretched to thin" feeling, but it is a shame, because this is a very interesting project!

That being said, I have to admit, that for my current application, I created a new "regular" Blazor Web App, and manually moved the "spark stuff" that I liked/needed.

Along the way, I also discovered the "scaffolding" feature of Visual Studio https://www.youtube.com/watch?v=tSUwr-R1JPQ

Which made me realize that even a fully supported "Spark" or "Spark-like" project would still "get stale" quickly/eventually... "Full templates" are a lot to "keep current".  And even a "perfect template" can only be used at the beginning of a project.  Making me realize that I wished all the Spark features were all in the the Spark commands... sort of like the Scaffolding tool. (I realized this before I discovered the Scaffolding tool... but since I had given it so much thought already... the Scaffolding tool immediately appealed to me.)

Which lead me to think that it would be nice to have the "Spark features" as "scaffolding options" that I could add as needed to my projects and maybe this is a direction that Wes could take Spark in.

Anyway, while I am a bit disillusioned and frustrated, I am going to "stick around" a bit longer, because Wes really does seem to be an interesting guy, with a deep understanding of Blazor and web development in general, so I am hoping/betting that he will "make a comeback" .
